vendor/dannyvankooten/vat.php/src/Rates/Clients/JsonVat.php

```diff
@@ -24,6 +24,8 @@ public function fetch() {
         curl_setopt($curl_handle, CURLOPT_URL, $url);
         curl_setopt($curl_handle, CURLOPT_CONNECTTIMEOUT, 10);
         curl_setopt($curl_handle, CURLOPT_RETURNTRANSFER, true);
+        // Added the following line to fix error due to missing local certificate
+        curl_setopt($curl_handle, CURLOPT_SSL_VERIFYPEER, false);
         $response_body = curl_exec($curl_handle);
         curl_close($curl_handle);
 
```

Comment:
I assume this will be overwritten when a 'composer update' is run. 
Worth sending a pull-request to the original vendor/author ?

Reply:
There is already a new version, but this one doesn't support php 5.6 anymore.
And on the other hand, this is more or less a quick 'n dirty fix. The "official" way would be to install local certificates...